cfgov/cfgov/settings/base.py

```diff
@@ -377,27 +377,18 @@
     "search.elasticsearch_helpers.WagtailSignalProcessor"
 )
 
-# S3 Configuration
-# https://django-storages.readthedocs.io/en/latest/backends/amazon-S3.html#settings
-AWS_LOCATION = "f"  # A path prefix that will be prepended to all uploads
-AWS_QUERYSTRING_AUTH = False  # do not add auth-related query params to URL
-AWS_S3_FILE_OVERWRITE = False
-AWS_S3_SECURE_URLS = True  # True = use https; False = use http
-AWS_STORAGE_BUCKET_NAME = os.environ.get("AWS_STORAGE_BUCKET_NAME")
-AWS_DEFAULT_ACL = None  # Default to using the ACL of the bucket
-
-if os.environ.get("S3_ENABLED", "False") == "True":
-    AWS_ACCESS_KEY_ID = os.environ["AWS_ACCESS_KEY_ID"]
-    AWS_SECRET_ACCESS_KEY = os.environ["AWS_SECRET_ACCESS_KEY"]
-    AWS_S3_CUSTOM_DOMAIN = os.environ.get("AWS_S3_CUSTOM_DOMAIN")
-    MEDIA_URL = os.path.join(
-        AWS_STORAGE_BUCKET_NAME + ".s3.amazonaws.com", AWS_LOCATION, ""
-    )
-
+if os.getenv("S3_ENABLED"):
```

Comment:
The previous code uses this logic:

```py
if os.environ.get("S3_ENABLED", "False") == "True":
```

but our internal deployment code only ever sets this environment variable if we use S3. I've thus simplified the logic here (and in `.env_SAMPLE`) so if this variable is defined in any way (e.g. `S3_ENABLED=1`), we enable S3.